### PR TITLE
Shorten action.yml description for marketplace

### PR DESCRIPTION
**Pull request by jwpereira**
Attempting to get into the marketplace now (woooo 🥳 ). First thing they flag is our descriptions being too long (needs to be < 125 characters). So, I've just grabbed the first sentence since it's straight to the point (the magic is really all @mw-hrastega).

As for the lack of punctuation at the end: that seems to be the trend with other actions 🤔

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -2,13 +2,7 @@
...
 name: Run MATLAB Command
 description: >-
-  Execute a MATLAB script, function, or statement. 
-  MATLAB exits with exit code 0 if the specified script, function, or statement
-  executes successfully without error. Otherwise, MATLAB terminates with a
-  nonzero exit code, which causes the build to fail. You can use the assert or
-  error functions in the command to ensure that builds fail when necessary. When
-  you use this task, all of the required files must be on the MATLAB search
-  path.
+  Execute a MATLAB script, function, or statement
```
Comment (mw-hrastega) [inline action.yml:5]: Run MATLAB scripts, functions, and statements

**Review by mw-hrastega — CHANGES_REQUESTED** [review]
Suggesting some minor edits

File: action.yml (hunk 1 above)
Comment (acampbel) [reply action.yml:5]: Houman, you're the maestro

**Review by acampbel — APPROVED** [review]

**Review by mcafaro — APPROVED** [review]
+1 Houman

**Review by mw-hrastega — APPROVED** [review]

**Review by acampbel — APPROVED** [review]

**Pull request merged**